src/hooks/useCallWithGasPrice.ts

```diff
@@ -0,0 +1,38 @@
+import { useCallback } from 'react'
+import ethers, { Contract, CallOverrides } from 'ethers'
+import { useGasPrice } from 'state/user/hooks'
+import { get } from 'lodash'
+
+/**
+ * Perform a contract call with a gas price returned from useGasPrice
+ * @param contract Used to perform the call
+ * @param methodName The name of the method called
+ * @param methodArgs An array of arguments to pass to the method
+ * @param overrides An overrides object to pass to the method
+ * @returns https://docs.ethers.io/v5/api/providers/types/#providers-TransactionReceipt
+ */
+export function useCallWithGasPrice() {
```

Comment:
Function used throughout application to call contract with user settings gasPrice